lib/tokenizers.js

```diff
@@ -83,7 +89,14 @@ export async function getCurrent() {
   }
 
   // 2. Get the current tokenizer.
-  return _getCurrentTokenizer();
+  let tokenizer;
+  try {
+    tokenizer = await _getCurrentTokenizer();
+  } catch(cause) {
+    throw new BedrockError('Could not get current tokenizer.', 'NotFoundError',
```

Comment:
```suggestion
    throw new BedrockError(
      'Could not get current tokenizer.', 'NotFoundError',
```

Reply:
Is this a not found error? I think it should be an `InvalidStateError` with a 503 -- since service will be unavailable without the current tokenizer.

Reply:
Looking at this more closely, We should just wrap this whole function in a try-catch and then emit this error.

Reply:
Changed to 503/InvalidStateError, added logging.

Reply:
@dlongley 
> Looking at this more closely, We should just wrap this whole function in a try-catch and then emit this error.

Not sure I understand -- like, wrap the contents of the function in a try-catch? Or somehow wrap its invocation?